utility/inputfile.cpp

```diff
@@ -67,11 +67,13 @@
 // Qt
 #include <QLoggingCategory>
 #include <QString>
+#include <QStringLiteral>
 #include <QTextStream>
+#include <qstringconverter_base.h> // QStringConverter
```

Comment:
`QStringConverterBase` is undocumented and should be considered an implementation detail
```suggestion
#include <QStringConverter>
```

Reply:
This is another weird one. If I include `<QStringConverter>` I get both a "QStringConverter is not used directly" for the header and then a "no header providing "QStringConverter" is directly included" for the function call later. If I include the base `.h` in the commit, it works fine. Same problem if I use `<QStringConverterBase>`. Ideas?

Reply:
Hmm, something worth bugging Qt about? `QStringConverterBase` shouldn't include `qstringcontainer.h`...

Reply:
https://bugreports.qt.io/browse/QTBUG-136076